### ISSUE TITLE
Publish blackbox and iOS starter app on deployment

### ISSUE THREAD

**Issue by domdom82** [openwhisk/openwhisk#560]
- [x] move blackbox to open
- [x] provide ansible playbooks to build and publish SDK artifacts
- [x] move travis build to ansible

**Comment by houshengbo** [comment]
Well, one more remark from me:
there are already issues like https://github.com/openwhisk/openwhisk/issues/502 and https://github.com/openwhisk/openwhisk/issues/74, opened nearly one week ago, and there are folks assigned to fix them with pull requests. The only thing that the assignees do not know is that we are moving from ant to ansible. 
It is not good that folks are doing things conflicting with each other. Maybe there is a lack of communication, but I hope this is something we need to improve in future.

**Comment by domdom82** [comment]
You are right. My original intent was to get our deployment to work on ansible. We were very far into development when we noticed blackbox and ios sdks were missing. We did not stumble across this earlier because there was no test for their existence. When I finally got my PR ready I was asked to move blackbox to open, which I did.

We just announced the shift to ansible today, however there will be a grace period of a few weeks where both ant and ansible deployments are supported as we iron out all of the little cracks and tweaks we have in the mess that is ant right now. So your work is not in vain, we still need ant for the time being.

I also agree PRs should not stay open for so long.
